cmd/kube-scheduler/app/options/deprecated.go

```diff
@@ -30,11 +30,11 @@ type DeprecatedOptions struct {
 	componentbaseconfig.ClientConnectionConfiguration
 	// Note that only the deprecated options (lock-object-name and lock-object-namespace) are populated here.
 	componentbaseconfig.LeaderElectionConfiguration
-	// PodMaxUnschedulableQDuration is the maximum time a pod can stay in
-	// unschedulableQ. If a pod stays in unschedulableQ for longer than this
-	// value, the pod will be moved from unschedulableQ to backoffQ or activeQ.
+	// PodMaxInUnschedulablePodsDuration is the maximum time a pod can stay in
+	// unschedulablePods. If a pod stays in unschedulablePods for longer than this
+	// value, the pod will be moved from unschedulablePods to backoffQ or activeQ.
 	// If this value is empty, the default value (60s) will be used.
```

Comment:
```suggestion
	// If this value is empty, the default value (5min) will be used.
```
Hi, I came from #108924. Has it been fixed in your local?